tests/methods/aci-18/input.dat

```diff
@@ -0,0 +1,66 @@
+#! Generated using commit GITCOMMIT
+#! Perform aci on benzyne 
+
+import forte
+
+memory 500 mb
+
+refscf    = -229.20378006852584 
+refaci    = -229.359450812283
+refacipt2 = -229.360444943286
+
+molecule mbenzyne{
+  0 1
+   C   0.0000000000  -2.5451795941   0.0000000000
+   C   0.0000000000   2.5451795941   0.0000000000
+   C  -2.2828001669  -1.3508352528   0.0000000000
+   C   2.2828001669  -1.3508352528   0.0000000000
+   C   2.2828001669   1.3508352528   0.0000000000
+   C  -2.2828001669   1.3508352528   0.0000000000
+   H  -4.0782187459  -2.3208602146   0.0000000000
+   H   4.0782187459  -2.3208602146   0.0000000000
+   H   4.0782187459   2.3208602146   0.0000000000
+   H  -4.0782187459   2.3208602146   0.0000000000
+
+  units bohr
+}
+
+set globals{
+   basis                   DZ
+   df_basis_mp2            cc-pvdz-ri
+   reference               uhf 
+   scf_type                pk
+   d_convergence           10
+   e_convergence           12
+   guess gwh
+}
+
+set forte {
+  multiplicity 1
+  frozen_docc            [2,1,0,0,0,0,2,1]
+  restricted_docc	  [3,2,0,0,0,0,2,3]
+  active                 [1,0,1,2,1,2,1,0]
+  root_sym               0
+  aci_nroot                  1
+  job_type               aci
+  sigma	                  0.001
+  aci_select_type            aimed_energy	 
+  aci_enforce_spin_complete  true
+  aci_add_aimed_degenerate false
+  aci_project_out_spin_contaminants false
+  diag_algorithm full
+  aci_batched_screening true
+  aci_nbatch 2
+#  aci_scale_sigma 0.0
+  active_guess_size 100
+}
```

Comment:
- [ ] Can we cleanup this input so that inputs are aligned and unused keywords removed?
- [ ] Can we add another test so we can check for more than 2 bins and/or multiple threads?